### PR TITLE
Carlos edits

### PR DESCRIPTION
**Pull request by cdelcastillo21**
Merging in ADCIRC Use Case Docs.

### CODE REVIEW

File: docs/dawson/usecase.md
Comment (kks32) [inline docs/dawson/usecase.md]:
```suggestion
The following use case presents an example of how to leverage the Tapis API to run an ensemble of HPC simulations. The specific workflow to be presented consists of running ADCIRC, a storm-surge modeling application available on DesignSafe, using the parametric job launcher pylauncher. All code and examples presented are meant to be be executed from a Jupyter Notebook on the DesignSafe platform and using a DesignSafe account to make Tapis API calls. Accompanying notebooks for this use case can be found in the ADCIRC folder in [Community Data](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/ADCIRC).
```

File: docs/dawson/usecase.md
Comment (kks32) [inline docs/dawson/usecase.md]:
```suggestion
* This software is distributed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.html).  
```

File: docs/dawson/usecase.md
Comment (kks32) [inline docs/dawson/usecase.md:40]: Could you please clarification on where this should be run with some screenshots?

File: docs/dawson/usecase.md
Comment (kks32) [inline docs/dawson/usecase.md]:
```suggestion
Check out the application from the github page - [https://github.com/UT-CHG/tapis-pylauncher](https://github.com/UT-CHG/tapis-pylauncher) and deploy it using either the Tapis CLI or AgavePy (See documentation links above under Tapis section), or email cdelcastillo21@gmail.com for access to an already deployed version of the application (it is not a public application yet, so has to be shared explicitly with users).
```

Comment (kks32) [reply docs/dawson/usecase.md]: We should discuss the best way to provide access to the App.

File: docs/dawson/usecase.md
Comment (kks32) [inline docs/dawson/usecase.md]:
```suggestion
The corresponding TACC base path to your project with a particular id can be found at `/corral-repl/projects/NHERI/projects/[id]/`. To find the ID for your project, you can just look at the URL of your project directory in designsafe:
```

**Review by kks32 — CHANGES_REQUESTED** [review]
Could you please review the whole document and see which sections are useful/can be trimmed in terms of code snippets.

File: docs/dawson/usecase.md
Comment (cdelcastillo21) [reply docs/dawson/usecase.md]: Yeah let me know what you think. Easiest way would be to make it a public app. Because for a user to deploy the app themselves they have to open a ticket to get app deployment privileges, and its a mess.

File: docs/dawson/usecase.md
Comment (cdelcastillo21) [reply docs/dawson/usecase.md:40]: I've added an image of executing the `tapis auth init --interactive` call from within a terminal in jupyter.

**Comment by cdelcastillo21** [comment]
With regards to shortening the document. I think it may make sense to wait until my colleague Mark Loveland reviews this documentation and tries to go through it to see what parts he thinks are too cumbersome. I agree there is a lot and it needs to be shortened, but I would like a second pair of eyes to help me determine from where it makes most sense to cut things (and where clarification is missing also).

**Review by kks32 — CHANGES_REQUESTED** [review]
Remove collaborator.png
Provide me access to edit please

**Review by kks32 — APPROVED** [review]

**Pull request merged**
